pandas/tseries/base.py

```diff
@@ -382,14 +393,18 @@ def argmin(self, axis=None):
             i8[mask] = np.iinfo('int64').max
         return i8.argmin()
 
-    def max(self, axis=None):
+    def max(self, axis=None, *args, **kwargs):
         """
-        return the maximum value of the Index
+        Return the maximum value of the Index or maximum along
```

Comment:
we RARELY copy-paste things. almost all doc strings are generated (esp/including this one)